var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -108,10 +159,12 @@ class Trilinos(CMakePackage):
     depends_on('glm')
     depends_on('metis@5:', when='+metis')
     depends_on('suite-sparse', when='+suite-sparse')
+    depends_on('zlib', when="+zlib")
 
     # MPI related dependencies
     depends_on('mpi')
     depends_on('netcdf+mpi')
+    depends_on('parallel-netcdf', when="+pnetcdf@master")
```

Comment:
is `parallel-netcdf` not available in any release versions of trilnos? is it available in `develop` branch?